mythril/laser/ethereum/strategy/extensions/bounded_loops.py

```diff
@@ -64,22 +64,30 @@ def get_strategic_global_state(self) -> GlobalState:
             else:
                 annotation = annotations[0]
 
-            address = state.get_current_instruction()["address"]
+            cur_instr = state.get_current_instruction()
 
-            if address in annotation._reached_count:
-                annotation._reached_count[address] += 1
+            if cur_instr["opcode"].upper() != "JUMPDEST":
+                return state
+
+            # create unique instruction identifier
+            key = "{}{}{}".format(
```

Comment:
```suggestion
            key = "{};{};{}".format(
```
Address and prev_pc are both integers, I believe that without delimiters there is a slight chanche for collision

Reply:
Oh! Yeah :rofl:

Reply:
Nice, thanks for the suggestion! Changed!